### ISSUE TITLE
Is there any convenient way that I can color my own generated character model?

### ISSUE THREAD

**Issue by Gloria-xhy** [kcyt/IntegratedPIFu#11]
Thanks for your apealling work,sir.The character model being generated has beyond my preliminary imagination.It certainly will be way more better if I am capable of coloring it.
**So I wanna ask that is there any convenient way that I can color my own generated character model or get the model texture and apply to it?**I would be very grateful if you could answer.Thanks

**Comment by kcyt** [comment]
In the code for "train_integratedPIFu.py", we have the following:
`color = index(image_tensor, uv).detach().cpu().numpy()[0].T`
`color = color * 0.5 + 0.5`
`save_obj_mesh_with_color(save_path, verts, faces, color)`
that projects a RGB image's colors onto the generated mesh. 

The RGB image is front view of a human subject. If you can train a model to predict the back view of the human subject, you can use similar code (as above) to 'color' the generated mesh. 

Another more established way of coloring a mesh is shown in PIFu (https://github.com/shunsukesaito/PIFu), where they use PIFu for texture prediction.
